pandas/_libs/groupby_helper.pxi.in

```diff
@@ -837,8 +823,3 @@ def group_cummax(ndarray[groupby_t, ndim=2] out,
                         if val > mval:
                             accum[lab, j] = mval = val
                         out[i, j] = mval
-
-
-group_cummax_float64 = group_cummax["float64_t"]
```

Comment:
are these just relics?

Reply:
groupby.ops (I think) looks these up by doing something like:

```
funcname = "group_cummax"
func = getattr(libgroupby, funcname, None)
if func is None:
    func = getattr(libgroupby, funcname+"_"+dtype)
```

Before we used the fused types, it would succeed on the second `getattr`.  Now it succeeds on the first.